SPECS/openssh/openssh.spec

```
@@ -60,6 +61,7 @@ This provides the ssh server daemons, utilities, configuration and service files
 %setup -q
 tar xf %{SOURCE1} --no-same-owner
 %patch0
+%patch101 -p1
```

Associated with above comment.
```suggestion
%patch1 -p1
```

---

same response as above; the patch number doesn't matter...